appcenter-data/ios/AppCenterReactNativeData/AppCenterReactNativeData.m

```diff
@@ -103,63 +195,85 @@ + (void)register {
                   writeOptions:(NSDictionary *)writeOptionsMap
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
-    MSWriteOptions *writeOptions = [self getWriteOptions:writeOptionsMap];
+    MSWriteOptions *writeOptions = getWriteOptions(writeOptionsMap);
     MSDictionaryDocument *document = [[MSDictionaryDocument alloc] initFromDictionary:documentMap];
     [MSData replaceDocumentWithID:documentID
                          document:document
                         partition:partition
                      writeOptions:writeOptions
-                completionHandler:[self dataCompletionHandler:kMSReplaceFailedErrorCode resolve:resolve reject:reject]];
+                completionHandler:dataCompletionHandler(kMSRemoveFailedErrorCode, resolve, reject)];
 }
 
 RCT_EXPORT_METHOD(remove:(NSString *)documentID
                   partition:(NSString *)partition
-                  writeOptions:(NSDictionary *)writeOptionsMap
+                  writeOptions:(NSDictionary* )writeOptionsMap
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
-    MSWriteOptions *writeOptions = [self getWriteOptions:writeOptionsMap];
+    MSWriteOptions *writeOptions = getWriteOptions(writeOptionsMap);
     [MSData deleteDocumentWithID:documentID
                        partition:partition
                     writeOptions:writeOptions
-               completionHandler:[self dataCompletionHandler:kMSRemoveFailedErrorCode resolve:resolve reject:reject]];
+               completionHandler:dataCompletionHandler(kMSRemoveFailedErrorCode, resolve, reject)];
+}
+
+- (void)closeList:(NSString *)paginatedDocumentsId {
+    [_paginatedDocuments removeObjectForKey:paginatedDocumentsId];
 }
 
-- (void (^)(MSDocumentWrapper * _Nonnull))dataCompletionHandler:(NSString *)errorCode
-                                                        resolve:(RCTPromiseResolveBlock)resolve
-                                                         reject:(RCTPromiseRejectBlock)reject{
-    return ^(MSDocumentWrapper * _Nonnull documentWrapper) {
+static void (^dataCompletionHandler(NSString* errorCode,
```

Comment:
In Objective-C, "static" method is used without `static` keyword but `+` sign.

```
+ (void)myStaticMethod
```

Reply:
Changed